### ISSUE TITLE
VSCode Insiders on Windows: Taskbar and StartMenu items started being back to the old orange icon

### ISSUE THREAD

**Issue by bergmeister** [Microsoft/vscode#59553]
Using the latest version from today, the taskbar and startmenu icons started to be orange whilst all other icons are still the expected green ones.
- VSCode Version:
```
Version: 1.28.0-insider (user setup)
Commit: 0bcd086e536f55b4bf608b64f7f8882e61d957ba
Date: 2018-09-27T10:09:19.780Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```
- OS Version: Windows 10 1803

![image](https://user-images.githubusercontent.com/9250262/46148674-7f1c2500-c260-11e8-9caf-9fe8dab1fed1.png)

![image](https://user-images.githubusercontent.com/9250262/46148829-d7ebbd80-c260-11e8-8918-7c3530dd7812.png)

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by bergmeister** [comment]
I checked and this is not a duplicate of the above issue suggested by the bot

**Comment by bpasero** [comment]
@misolori maybe you also check, I thought my fix went through.

**Comment by bpasero** [comment]
Seems to be fixed now with latest:

![image](https://user-images.githubusercontent.com/900690/46154265-14c1af80-c275-11e8-84ed-c49c6067d12c.png)

**Comment by misolori** [comment]
Also verified that it is now fixed:
<img width="25%" src="https://user-images.githubusercontent.com/35271042/46154571-524d1b00-c22a-11e8-80b8-f7d2f17a1cf2.png" />
<img width="25%" src="https://user-images.githubusercontent.com/35271042/46154580-55e0a200-c22a-11e8-8682-06ba018b5679.png" />
<img width="50%" src="https://user-images.githubusercontent.com/35271042/46154600-5bd68300-c22a-11e8-8218-af337a998956.png" />
<img width="50%" src="https://user-images.githubusercontent.com/35271042/46154648-76a8f780-c22a-11e8-957e-3d3d400a0ac4.png" />

**Comment by bergmeister** [comment]
My exe had the green icon as well but only after a reboot, the icon changed back to the green one. This seems to me to be a persistence bug in Windows itself.
